CODEOWNERS

```diff
@@ -402,6 +402,7 @@ Makefile* @cilium/build
 /operator/pkg/networkpolicy @cilium/sig-policy
 /operator/pkg/secretsync @cilium/sig-servicemesh
 /operator/watchers/bgp.go @cilium/sig-bgp
+/pkg/act/ @cilium/sig-datapath
```

Comment:
maybe @cilium/metrics should additionally own this?

Reply:
What do you think about moving it under `pkg/metrics`?